src/lib/Libpython/pbs_python_svr_internal.c

```diff
@@ -5252,14 +5255,27 @@ _pbs_python_event_set(unsigned int hook_event, char *req_user, char *req_host,
 		log_event(PBSEVENT_DEBUG3, PBS_EVENTCLASS_HOOK, LOG_INFO, __func__, log_buffer);
 	}
 
+	lval = enable_restart;
+	if (is_sattr_set(SVR_ATR_PythonRestartEnable))
+		enable_restart = get_sattr_long(SVR_ATR_PythonRestartEnable);
+	else
+		enable_restart = PBS_PYTHON_RESTART_ENABLE;
+	if (lval != enable_restart) {
+		snprintf(log_buffer, sizeof(log_buffer),
+			"python_enable_restart is now %ld", enable_restart);
```

Comment:
log correct option name "python_restart_enable" .

Reply:
There is a logging function call log_eventf() which combines both sprintf() and log_event().  Consider using it instead of creating a string to just print.